chart/README.md

```diff
@@ -208,10 +208,14 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `scheduler.serviceAccountAnnotations`                 | Annotations to add to scheduler kubernetes service account                                                   | `{}`                                            |
 | `scheduler.extraVolumes`                              | Mount additional volumes into scheduler                                                                      | `[]`                                            |
 | `scheduler.extraVolumeMounts`                         | Mount additional volumes into scheduler                                                                      | `[]`                                            |
+| `webserver.livenessProbe.path`                        | Webserver LivenessProbe path (need to match your webserver base_url)                                         | `/health`                                              |
+| `webserver.livenessProbe.host`                        | Webserver LivenessProbe host header (need to match your webserver base_url)                                  | `~`                                              |
 | `webserver.livenessProbe.initialDelaySeconds`         | Webserver LivenessProbe initial delay                                                                        | `15`                                              |
 | `webserver.livenessProbe.timeoutSeconds`              | Webserver LivenessProbe timeout seconds                                                                      | `30`                                              |
 | `webserver.livenessProbe.failureThreshold`            | Webserver LivenessProbe failure threshold                                                                    | `20`                                              |
 | `webserver.livenessProbe.periodSeconds`               | Webserver LivenessProbe period seconds                                                                       | `5`                                               |
+| `webserver.readinessProbe.path`                       | Webserver ReadinessProbe path (need to match your webserver base_url)                                         | `/health`                                              |
+| `webserver.readinessProbe.host`                       | Webserver ReadinessProbe host header (need to match your webserver base_url)                                  | `~`                                              |
```

Comment:
Do we ever need different values here -- they are controlled by the same airflow.cfg so these two values should always be the same, right?

Reply:
Yes, the values will be the same, and they need to be "in sync" also with `.webserver.base_url` in `airflow.cfg`. So now I changed so that it extract the host and path from the `webserver.base_url` if provided.